docs/streams/developer-guide/config-streams.html

```diff
@@ -269,6 +269,11 @@ <h4><a class="toc-backref" href="#id5">bootstrap.servers</a><a class="headerlink
               <td colspan="2">The amount of time in milliseconds, before a request is retried. This applies if the <code class="docutils literal"><span class="pre">retries</span></code> parameter is configured to be greater than 0. </td>
               <td>100</td>
           </tr>
+          <tr class="row-even"><td>rocksdb.config.setter</td>
```

Comment:
Just a question: what is the `row-even` vs `row-odd` tag? Seems we don't use is consistently (there are multiple `row-even` tags in a row above) and it does also not seem to make any difference how the rows are rendered? Why do we even have this tag?

Reply:
Good question, I added it just keep the style consistent.